src/components/Persisted/Persisted.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Avatar, RaisedButton } from 'material-ui';
+import { Link } from 'react-router-dom';
+
+class Persisted extends Component {
+  panic = () => {
+    // use twilio API to text the mental health hotline
+  };
+
+  render () {
+    const { currentUser, signOut } = this.props;
+    const { pathname } = this.props.location;
+    const userKeys = Object.keys(currentUser);
+
+    const userInfo = <div>
+      {/* placeholder avatar, should fetch from server?  */}
+      <Avatar src="https://speakerdata2.s3.amazonaws.com/photo/image/839843/thats-what-she-said-0413-1-lgn.jpg" alt="Your user photo"/>
```

Comment:
I'm embarrassed I had to open this to find out what it was going to be.

Reply:
I am also embarrassed on your behalf. You know me better than that. P.S. I got a giant poster of Kate for Christmas, and I'm hanging it over the mantel in the TV room AT THIS EXACT MOMENT.